src/js/view/controls/components/timeslider.js

```diff
@@ -177,7 +177,7 @@ class TimeSlider extends Slider {
             return;
         }
         this.reset();
-        this.updateCues(model, model.get('cues'));
+        this.updateCues(model, []);
```

Comment:
Just update the model and let the `change` callback handle `updateCues`"
 
`model.set('cues', []);`